content/en/tracing/profiler/enabling/node.md

```diff
@@ -0,0 +1,73 @@
+---
+title: Enabling the Node.js Profiler
+kind: Documentation
+code_lang: node
+type: multi-code-lang
+code_lang_weight: 40
+further_reading:
+    - link: 'getting_started/profiler'
+      tag: 'Documentation'
+      text: 'Getting Started with Profiler'
+    - link: 'tracing/profiler/search_profiles'
+      tag: 'Documentation'
+      text: 'Learn more about available profile types'
+    - link: 'tracing/profiler/profiler_troubleshooting'
+      tag: 'Documentation'
+      text: 'Fix problems you encounter while using the profiler'
+---
+
+<div class="alert alert-warning">
+Datadog Node.js Profiler is currently in public beta. Datadog recommends evaluating the profiler in a non-sensitive environment before deploying in production.
+</div>
+
+The profiler is shipped within Datadog tracing libraries. If you are already using [APM to collect traces][1] for your application, you can skip installing the library and go directly to enabling the profiler.
+
+## Requirements
+
+The Datadog Profiler requires Node.js 12+.
```

Comment:
This seems like it might be a bit much for a ~quickstart guide. Could we say something like
> The Datadog Profiler requires Node.js 12+, but Node.js 16+ is recommended. Some applications will see increased tail latency in between profiles on Node.js < 16. See <link> for more details.

cc @GaurabAryal @DataDog/documentation if you have any suggestions for better phrasing here!